docs/user_guides/single_table/predefined_constraints.rst

```diff
@@ -0,0 +1,410 @@
+
+.. ipython:: python
+    :okwarning:
+
+    def years_in_the_company(data):
+        return data['age'] - data['age_when_joined']
+
+Once we have defined this function, we can use the ``ColumnFormula``
+constraint by passing it:
+
+-  the name of the column that we want to generate
+-  the function that generates the column values
+-  the handling strategy that we want to use
+
+.. ipython:: python
+    :okwarning:
+
+    from sdv.constraints import ColumnFormula
+
+    years_in_the_company_constraint = ColumnFormula(
+        column='years_in_the_company',
+        formula=years_in_the_company,
+        handling_strategy='transform'
+    )
+
+Rounding Constraint
+~~~~~~~~~~~~~~~~~~~
+
+In order for data to be realistic, we also might want to round data
+to a certain number of digits. To do this, we can use the Rounding
+Constraint. We will pass this constraint:
+
+-  the name of the column(s) that should be rounded.
+-  the number of digits each column should be rounded to.
+-  the handling strategy that we want to use
+-  (optional) if reject sampling, we can customize the threshold of
+   the sampled values.
+
+.. ipython:: python
+    :okwarning:
+
+    from sdv.constraints import Rounding
+
+    salary_rounding_constraint = Rounding(
+        columns='salary',
+        digits=2,
+        handling_strategy='transform'
+    )
+
+Between Constraint
+~~~~~~~~~~~~~~~~~~
+
+Another possibility is the ``Between`` constraint. It guarantees
+that one column is always in between two other columns/values. For example,
+the ``age`` column in our demo data is realistically bounded to the ages of
+15 and 90 since acual employees won't be too young or too old.
+
+In order to use it, we need to create an instance passing:
+
+-  the name of the ``low`` column or a scalar value to be used as the lower bound
+-  the name of the ``high`` column or a scalar value to be used as the upper bound
+-  the handling strategy that we want to use
+
+.. ipython:: python
+    :okwarning:
+    
+    from sdv.constraints import Between
+
+    reasonable_age_constraint = Between(
+        column='age',
+        low=15,
+        high=90,
+        handling_strategy='transform'
+    )
+
+OneHotEncoding Constraint
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Another constraint available is the ``OneHotEncoding`` constraint.
+This constraint allows the user to specify a list of columns where each row 
+is a one hot vector. Then, the constraint will make sure that the output
+of the model is transformed so that the column with the largest value is
+set to 1 while all other columns are set to 0. To apply the constraint we
+need to create an instance passing:
+
+- A list of the names of the columns of interest
+- The strategy we want to use (``transform`` is recommended)
+
+.. ipython:: python
+    :okwarning:
+
+    from sdv.constraints import OneHotEncoding
+
+    one_hot_constraint = OneHotEncoding(
+        columns=['full_time', 'part_time', 'contractor']
+    )
+
+Multi-row Constraints
```

Comment:
Suggestions: I find it difficult to follow how is "multi-row" different from the previous constraints because in the previous constraints, we do not explicitly mention they are "column-based" or whatever. I would either remove it or include "Multi-column Constraints" or simply "Row Constraints" and "Column Constraints" headers instead.